portfolio/src/main/webapp/index.html

```diff
@@ -4,15 +4,16 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <script type="text/javascript" src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Jack's Main Portfolio</h1>
+      <img src="images/headshot.jpg" style="width:128px;height:128px;"/>
```

Comment:
It's good practice to add an `alt` attribute to describe the content if it fails to load or for people using screen readers.

https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback